src/test/java/duke/task/DeadlineTest.java

```diff
@@ -0,0 +1,22 @@
+package duke.task;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+
+
+public class DeadlineTest {
+    @Test
+    public void addToDoTest() {
```

Comment:
Minor naming issue, but maybe the test method name can be simplified to `addToDo()`, as the JUnit annonation does suggest it's a test method. Just reads more cleanly